src/app/layout.jsx

```
@@ -35,26 +38,40 @@ const modeScript = `
+export const metadata = {
+  title: 'AOSSIE',
+  description: 'AOSSIE Website',
+}
+
+export default function RootLayout({ children }) {
   return (
-    <Html className="h-full antialiased" lang="en">
-      <Head>
+    <html className="h-full antialiased" lang="en" suppressHydrationWarning>
+      <head>
         <script dangerouslySetInnerHTML={{ __html: modeScript }} />
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)